scss/components/_activity-panel.scss

```diff
@@ -1,16 +1,27 @@
 .civicase__activity-panel {
-  margin-top: 37px;
+  &.affix {
+    position: fixed !important;
+  }
 
   .panel {
     overflow: auto;
+    position: relative;
+  }
 
-    &.affix {
-      position: fixed !important;
-      width: calc(30% - 12px); // 12px = padding outside civicase__activity-feed__list-container * 30%
-    }
+  .panel-heading,
+  .panel-subheading {
+    position: absolute;
```

Comment:
this seems pretty unstable, why do you need the whole header with position absolute?

Reply:
The `.civicase__activity-panel`(which is the container for these headers), has fixed position. And inside that we need fixed position for the Header, Subheader and Footer. Thats why `position: absolute;` is added to apply stickiness.

Reply:
@reneolivo can you check if we can avoid setting `position: absolute` to these elements

Reply:
@reneolivo could you check if can avoid setting `position: absolute;` it makes the component very unstable i believe

Reply:
Removed it. It's not necessary.